tree/30_generic_methods/condition_from_command.cf

```diff
@@ -0,0 +1,59 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Condition from command 
+# @description Execute a command and create outcome classes depending on its exit code
+#
+# @parameter condition_prefix   The condition name, use ${class_prefix} to create a local condition
+# @parameter command            The command to run
+# @parameter true_codes         List of codes that produce a true status separated with commas (ex: 1,2,5)
+# @parameter false_codes        List of codes that produce a false status separated with commas (ex: 3,4,6)
```

Comment:
Yes, like every generic method does.
But the rest of the documentation details what else is done.